source/views/sis/course-search/search.js

```diff
@@ -141,16 +141,17 @@ function applyFiltersAndQuery(args: {
 	const filteredCourses = allCourses.filter(course =>
 		applyFilters(filters, course),
 	)
-	const grouped = groupBy(filteredCourses, r => r.term)
+	const sortedCourses = sortBy(filteredCourses, course => deptNum(course))
+	const grouped = groupBy(sortedCourses, r => r.term)
 	const groupedCourses = toPairs(grouped).map(([key, value]) => ({
 		title: key,
 		data: value,
 	}))
 
-	const sortedCourses = sortBy(groupedCourses, course => course.title).reverse()
+	const sortedTerms = sortBy(groupedCourses, course => course.title).reverse()
```

Comment:
Make sortedCourses a "let" instead, and reassign to it here

Reply:
sortedCourses and sortedTerms are pretty different though...sortedCourses is an array of courses, whereas sortedTerms is an array of terms (courses grouped into terms), do we want to store them in the same variable?

Reply:
Ah. I missed the grouping into terms, even though I read through to see if that's what you were doing. If the name's descriptive, let's keep it.